graylog2-web-interface/src/components/alertconditions/AlertConditionForm.jsx

```diff
@@ -8,26 +8,44 @@ const AlertConditionForm = React.createClass({
     type: React.PropTypes.string.isRequired,
     alertCondition: React.PropTypes.object,
   },
-  alertConditionsFactory: new AlertConditionsFactory(),
+  getDefaultProps() {
+    return {
+      alertCondition: {
+        parameters: {},
```

Comment:
I just realised that in 2.0, creating a new alert condition provides default values for parameters, and after these changes we don't do that any more. To fix that, we should remove the empty object from here, as each specific alert condition form provides some default props if the parameters are not given.